content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+
+Meskipun etcd menyimpan ID anggota unik secara internal, disarankan untuk menggunakan nama unik untuk setiap anggota untuk menghindari kesalahan manusia. Sebagai contoh, sebuah klaster etcd dengan tiga anggota. Jadikan URL-nya, member1=http://10.0.0.1, member2=http://10.0.0.2, and member3=http://10.0.0.3. Ketika member1 gagal, ganti dengan member4=http://10.0.0.4.
+
+1. Dapatkan ID anggota yang gagal dari member1:
+
+    `etcdctl --endpoints=http://10.0.0.2,http://10.0.0.3 member list`
+
+      Akan tampil pesan berikut:
+
+        8211f1d0f64f3269, started, member1, http://10.0.0.1:2380, http://10.0.0.1:2379
+        91bc3c398fb3c146, started, member2, http://10.0.0.2:2380, http://10.0.0.2:2379
+        fd422379fda50e48, started, member3, http://10.0.0.3:2380, http://10.0.0.3:2379
+
+2. Hapus anggota yang gagal:
+
+    `etcdctl member remove 8211f1d0f64f3269`
+
+      Akan tampil pesan berikut:
+
+       Removed member 8211f1d0f64f3269 from cluster
+
+3. Tambahkan anggota baru:
+
+    `./etcdctl member add member4 --peer-urls=http://10.0.0.4:2380`
+
+     Akan tampil pesan berikut:
+
+       Member 2be1eb8f84b7f63e added to cluster ef37ad9dc622a7c4
+
+4. Jalankan anggota yang baru ditambahkan pada mesin dengan IP `10.0.0.4`:
+
+        export ETCD_NAME="member4"
+        export ETCD_INITIAL_CLUSTER="member2=http://10.0.0.2:2380,member3=http://10.0.0.3:2380,member4=http://10.0.0.4:2380"
+        export ETCD_INITIAL_CLUSTER_STATE=existing
+        etcd [flags]
+
+5. Lakukan salah satu dari yang berikut:
+
+   1. Perbarui _flag_ `--etcd-server` untuk membuat Kubernetes mengetahui perubahan konfigurasi, lalu start ulang server API Kubernetes.
+   2. Perbarui konfigurasi _load balancer_ jika _load balancer_ digunakan dalam Deployment.
+
+Untuk informasi lebih lanjut tentang konfigurasi ulang klaster, lihat [Dokumentasi Konfigurasi etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member).
+
+## Mencadangkan klaster etcd
+
+Semua objek Kubernetes disimpan dalam etcd. Mencadangkan secara berkala data klaster etcd penting untuk memulihkan klaster Kubernetes di bawah skenario bencana, seperti kehilangan semua Node _control plane_. Berkas _snapshot_ berisi semua status Kubernetes dan informasi penting. Untuk menjaga data Kubernetes yang sensitif aman, enkripsi berkas _snapshot_.
+
+Mencadangkan klaster etcd dapat dilakukan dengan dua cara: _snapshot_ etcd bawaan dan _snapshot_ volume.
+
+### Snapshot bawaan
+
+etcd mendukung _snapshot_ bawaan, jadi mencadangkan klaster etcd lebih mudah. _snapshot_ dapat diambil dari anggota langsung dengan command `etcdctl snapshot save` atau dengan menyalin `member/snap/db` berkas dari etcd [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir) yang saat ini tidak digunakan oleh proses etcd. Mengambil _snapshot_ biasanya tidak akan mempengaruhi kinerja anggota.
```

Comment:
nit: open sentence with uppercase
```suggestion
Fitur _snapshot_ didukung oleh etcd secara bawaan, jadi mencadangkan klaster etcd lebih mudah. _Snapshot_ dapat diambil dari anggota langsung dengan command `etcdctl snapshot save` atau dengan menyalin `member/snap/db` berkas dari etcd [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir) yang saat ini tidak digunakan oleh proses etcd. Mengambil _snapshot_ biasanya tidak akan mempengaruhi kinerja anggota.
```

Reply:
Sorry, I forgot to uppercase this sentence.